EnglishTranslation-master/QUEST.tsv

```diff
@@ -1023,13 +1023,13 @@ QUEST_20150317_001022	I guess the real revelation is in the chapel? {nl}That Lai
 QUEST_20150317_001023	This is the Seal of Space...{nl}
 QUEST_20150317_001024	We will need the Revelator.
 QUEST_20150317_001025	The Revelator has came here alone. {nl}Exactly what I wanted.
-QUEST_20150317_001026	For what purpose did you come to this cursed city? {nl}Please go back if you don't have a particular purpose.
+QUEST_20150317_001026	For what purpose did you come to this cursed city? {nl}Please go back if you don't have a particular reason.
```

Comment:
To make a more natural sounding:

For what purpose did you come to this cursed city? {nl}Please go back if you have nothing particular to attend to.